datasets/CodeCompliantEquipment.idf

```diff
@@ -0,0 +1,2427 @@
+!---------------------------------------------------------------------------------------------
+!- This dataset includes performance curve objects that describe the part load performance of
+!- building energy code minimally compliant equipment.
+!-
+!- The dataset provides curves for the following EnergyPlus object:
+!-   * Chiller:Electric:EIR
+!-
+!- Sets of curves are provided for the following building energy codes minimum requirements:
+!-   * ASHRAE Standard 90.1-2019 Table 6.8.1-3
+!-   * ASHRAE Standard 90.1-2019 Table G3.5.3
+!-
+!- Last update: Sept. 2021
+!---------------------------------------------------------------------------------------------
+
+!----------------------------------------------------------
+!- Beginning of performance curves for Chiller:Electric:EIR:
+
+Curve:Quadratic,
+    ASHRAE901_AppG_wtr_cent_gte150lt300ton_0.634kWpton_0.596IPLV.IP_eir-f-plr,    !- Name
+    0.3208405333114065,       !- Coefficient1 Constant
+    0.03157995838320167,      !- Coefficient2 x
+    0.6470161436053915,       !- Coefficient3 x2
+    0.0,                      !- Minimum Value of x
+    1.0,                      !- Maximum Value of x
```

Comment:
This means the input PLR will be capped at 1.0, even if the chiller has a Maximum Part Load Ratio > 1.0. Is that ok here?

Reply:
That's our recommendation, and because the curves aren't normative, users could overwrite to extrapolate if need be.